test/abort/test-addon-uv-handle-leak.js

```diff
@@ -47,8 +47,8 @@ if (process.argv[2] === 'child') {
 
   // Parse output that is formatted like this:
 
-  // uv loop at [0x559b65ed5770] has active handles
-  // [0x7f2de0018430] timer
+  // uv loop at [0x559b65ed5770] has open handles:
+  // [0x7f2de0018430] timer (active)
```

Comment:
Maybe `// uv loop at [0x559b65ed5770] has 3 open handles in total` at the bottom? Just a suggestion.

Reply:
Ah yes, thanks for noticing :) Done!